### DISCUSSION TITLE
What are the save-txt values?

### DISCUSSION THREAD

**Discussion by Kevinaragon93** [mikel-brostrom/yolo_tracking#658] [Q&A] [answered]
What do the values of the tracking result txt stand for?
The first is the label frame and the four others are related to the bounding boxes, but what's their value exactly?

![Screenshot 2022-12-20 130234](https://user-images.githubusercontent.com/72398728/208735481-3eaf3204-2606-4b3c-88ea-5607f3fc85cd.png)

**Comment by mikel-brostrom** [comment, marked as answer]
The file format should be the same as the ground truth file, which is a CSV text-file containing one object instance per line. Each line must contain 10 values:

`<frame>, <id>, <bb_left>, <bb_top>, <bb_width>, <bb_height>, <conf>, <x>, <y>, <z> `

 The conf value contains the detection confidence in the det.txt files. For the ground truth, it acts as a flag whether the entry is to be considered. A value of 0 means that this particular instance is ignored in the evaluation, while any other value can be used to mark it as active. For submitted results, all lines in the .txt file are considered. The world coordinates x,y,z are ignored for the 2D challenge and can be filled with -1. Similarly, the bounding boxes are ignored for the 3D challenge. However, each line is still required to contain 10 values.

**Reply by Kevinaragon93** [reply]
Perfect, thank you soo much Mike for your help !
